config.toml

```diff
@@ -0,0 +1,85 @@
+[LOADED_NETWORK]
+loaded_network_directory = "Q:/Model Development/Simwrapper Validation/notebooks/roads/Data"
+
+# The dbf file name is LOAD[TOD].dbf by default
+loaded_network_time = ["AM", "MD", "PM", "EV", "EA"]
+
+# The needed CHAMP output columns
+loaded_network_col_names = ["A", "B", "AT", "FT", "V_1"]
+
+[OBSERVED]
+# The observed data file path
+obs_filepath = "Q:/Data/Observed/Validation/2019_old/observed_counts.csv"
+
+# The columns used for calculation, usually are 'A,B, and Counts'
+obs_usecols = ["A", "B", "Daily", "AM", "MD", "PM", "EV", "EA"]
+
+# The columns we want to include from the observed data, DONT CHANGE A, B!
+obs_extra_columns = ["A", "B", "Loc Type", "Source", "Source ID", "Location", "Dir"]
+
+[SCATTER_INPUT]
+# The time period we want to display in the scatter plot
+chosen_period = "Daily"
+
+# Change the variables to show different group_by result
+classification_col = "Loc Type"
+
+# cols displayed and information we want in our generated csv
+combined_df_cols = ["Loc Type", "A", "B", "AT", "FT", "Source ID", "AT Group", "FT Group", "Location", "Road", "Dir"]
+
+# YAML filename
+yaml_path = "dashboard{dashboard_num1}-Scatter.yaml"

```

Comment:
Is data embedded directly in the yamls?  It would be ideal to just have static yamls that we don't need to generate programmatically

Reply:
The data in the YAMLs is not embedded directly, but rather, the YAML files reference configuration files (such as .vega.json files) that define the actual visualizations or data settings. But I can avoid generating YAML in the script by setting a static one. Actually that's what I did in the statistic section in order to avoid error